modules/gdscript/editor/gdscript_docgen.cpp

```diff
@@ -340,8 +340,8 @@ void GDScriptDocGen::_generate_docs(GDScript *p_script, const GDP::ClassNode *p_
 	doc.script_path = p_script->get_script_path();
 
 	if (p_script->base.is_valid() && p_script->base->is_valid()) {
-		if (!p_script->base->doc.name.is_empty()) {
-			doc.inherits = p_script->base->doc.name;
+		if (!p_script->base->fully_qualified_name.is_empty()) {
+			doc.inherits = p_script->base->fully_qualified_name;
```

Comment:
This is an incorrect fix. The FQCN format does not always match the doc class name format.

Reply:
Would a check on `base->global_name` (or maybe `base->local_name` for inner classes) be better? and if empty, use `base->path`?

Reply:
I think the problem most likely lies somewhere outside. This code relies on the assumption that the base class documentation has already been built at this point (documentation generation happens when the script is compiled, as far as I remember). This is a valid assumption, so we need to figure out why it is not being met in this case.